Cilsil/Cil/Parsers/RetParser.cs

```diff
@@ -15,40 +15,25 @@ internal class RetParser : InstructionParser
         protected override bool ParseCilInstructionInternal(Instruction instruction,
```

Comment:
All the changes in this file is to un-modify RetParser as we do dummy returned expression